service/src/test/java/org/cbioportal/service/util/ProfiledCasesCounterTest.java

```diff
@@ -113,7 +113,7 @@ public void calculate() throws Exception {
         
         profiledSamplesCounter.calculate(
                 Arrays.asList(MOLECULAR_PROFILE_ID, MOLECULAR_PROFILE_ID, MOLECULAR_PROFILE_ID),
-                Arrays.asList(SAMPLE_ID_1, SAMPLE_ID_2, SAMPLE_ID_3), alterationCounts, true);
+                Arrays.asList(SAMPLE_ID_1, SAMPLE_ID_2, SAMPLE_ID_3), alterationCounts, true, false);
```

Comment:
@kalletlak might be good to add some tests here for groups with no alterations if possible? Or otherwise maybe file an issue that we need a E2E test for it? Thanks! I'm not sure how our suite of current group comparison e2e tests is looking, maybe we can easily add this one?

Reply:
added